bot/modules/nostr/events.js

```diff
@@ -31,8 +31,11 @@ exports.orderCreated = async order => {
   event.content = JSON.stringify(evData);
   if (order.community_id) {
     const community = await Community.findById(order.community_id);
-    if (community.public && community.nostr_public_key) {
-      event.tags.push(['p', community.nostr_public_key]);
+    if (community.public) {
+      if (community.nostr_public_key) {
+        event.tags.push(['p', community.nostr_public_key]);
+      }
+      event.tags.push(['com', community.group]);
```

Comment:
I think it would be better to add the community id, the group can be modified and in some cases can have some issues